packages/template-retail-react-app/app/components/product-view-modal/bundle.jsx

```diff
@@ -0,0 +1,146 @@
+                            )}
+                        </Box>
+
+                        <VStack align="stretch" flex={1}>
+                            {/* Parent product */}
+                            <Box marginBottom={6}>
+                                <ProductView
+                                    showFullLink={false}
+                                    showImageGallery={false}
+                                    product={productViewModalData.product}
+                                    isLoading={productViewModalData.isFetching}
+                                    updateCart={(product, quantity) =>
+                                        updateCart(product, quantity, selectedChildProducts)
+                                    }
+                                    validateOrderability={() => {
+                                        return Object.values(childProductRefs.current).every(
+                                            ({validateOrderability}) => validateOrderability()
+                                        )
+                                    }}
+                                    {...props}
+                                />
+                            </Box>
+
+                            {childProducts &&
+                                childProducts.data.map((_product, i) => {
+                                    const product = {
+                                        ..._product,
+                                        ...productViewModalData.product.bundledProductItems[i]
+                                    }
+                                    const quantityPerBundle = product.quantity / bundle.quantity
+
+                                    return (
+                                        <ProductView
+                                            key={i}
+                                            // Do not use an arrow function as we are manipulating the functions scope.
+                                            ref={function (ref) {
+                                                // Assign the "set" scope of the ref, this is how we access the internal validation.
+                                                childProductRefs.current[product.id] = {
+                                                    ref,
+                                                    validateOrderability: this.validateOrderability
+                                                }
+                                            }}
+                                            showImageGallery={false}
+                                            isProductPartOfBundle={true}
+                                            showFullLink={false}
+                                            product={product}
+                                            isLoading={isLoading}
+                                            childProductOrderability={childProductOrderability}
```

Comment:
`childProductOrderability` controls whether or not the `Update` button is disabled or not based on the orderability status of the child products. Since the `Update` button is attached to the parent and not the children, this should get passed as a prop in the parent `ProductView` component

Reply:
I see, thanks! The `childProductOrderability` is for the _parent_ product (specifically to enable/disable its Update button), while the set method for this prop is to be assigned to the _children_ products.